README.md

```diff
@@ -1 +1,108 @@
-🏴‍☠️
+# Node.js Feature flags
+
+## Usage
+
+### Environment
+
+You'll need a `LAUNCHDARKLY_SDK` environment variable configured, otherwise all
+flag queries will return false by default.
```

Comment:
true by default

Reply:
Good catch, I think it needs to be false by default now, since `getAllFlags` has no understanding of the possible flags, we just return an empty object, so those will all be nulls. Does that make sense? I do need to update the stub obj to reflect this.

Reply:
Sounds good to me